Dataset.py

```diff
@@ -694,6 +743,13 @@ def get_all_tags(self):
 
   def get_total_num_seqs(self):
     """
+    TODO: this function is only used in DataSet and MetaDataset, so can be protected
+      This function should either be removed, or given a useful purpose
+      In Dataset is only used for an assert when self.seq_tags_filter is not None
+      In MetaDataset, it is surrounded with a try that catches for NotImplemented,
+      and just continues without any exception.
+      Meaning, this function is completely useless right now, and not used at any relevant point
```

Comment:
This function is actually very important for any kind of meta dataset. I don't really understand why you think it's not important?